data/json/harvest.json

```diff
@@ -48,5 +48,33 @@
       { "drop": "sinew", "base_num": [ 50, 100 ], "scale_num": [ 0.6, 0.9 ], "max": 120 },
       { "drop": "raw_fur", "base_num": [ 2, 4 ], "scale_num": [ 0.6, 0.8 ], "max": 5 }
     ]
+  },
+  {
+    "id": "acidant_med",
+    "type": "harvest",
+    "message": "You carefully carve open the carapace, avoiding the acrid corrosion.",
+    "entries": [
+      { "drop": "meat_tainted", "base_num": [ 1, 5 ], "scale_num": [ 0.3, 0.7 ], "max": 8 },
+      { "drop": "acidchitin_piece", "base_num": [ 1, 4 ], "scale_num": [ 0.2, 0.5 ], "max": 6 }
+    ]
+  },
+  {
+    "id": "acidant_larva",
+    "type": "harvest",
+    "message": "You delicately cut open the soft tissue, avoiding the corroding fluids.",
+    "entries": [
+      { "drop": "meat_tainted", "base_num": [ 1, 3 ], "scale_num": [ 0.2, 0.5 ], "max": 4 },
+      { "drop": "acidchitin_piece", "base_num": [ 0, 1 ], "scale_num": [ 0.2, 0.5 ], "max": 1 }
+    ]
+  },
+  {
+    "id": "acidant_queen",
+    "type": "harvest",
+    "message": "You laboriously dissect the colossal insect.",
+    "entries": [
+      { "drop": "meat", "base_num": [ 1, 3 ], "scale_num": [ 0.3, 0.7 ], "max": 4 },
```

Comment:
Queen drops seem really low for a LARGE monster.

Reply:
You're right, looks like I copied over from _med and forgot to update that one.